nebu/config.py

```diff
@@ -83,10 +86,19 @@ def discover_settings():
         },
     }
     for section in config.sections():
-        if not section.startswith(CONFIG_SECTION_ENVIRON_PREFIX):
-            continue  # ignore all other sections
-        short_name = section[len(CONFIG_SECTION_ENVIRON_PREFIX):]
-        settings['environs'][short_name] = dict(config[section])
+        if section.startswith(CONFIG_SECTION_ENVIRON_PREFIX):
+            short_name = section[len(CONFIG_SECTION_ENVIRON_PREFIX):]
+            settings['environs'][short_name] = dict(config[section])
+        elif section == 'settings':
+            for k, v in config[section].items():
+                if ',' in v:
+                    settings[k] = [i.strip() for i in v.split(',')
+                                   if i.strip()]
```

Comment:
I chose to detect comma-separated list to process as a list, to avoid embedding knowledge of specific config names here. It _might_ be better to have a convention for option names (like `option.endswith('list')`) and split on whitespace - that's the natural separator for this particular key (class names are whitespace separated)

Or, just dump the string value in here, and do the split in the code where it is used. Might make more sense.

Reply:
I'm leaning toward the last, actually - let config users deal with the need-to-be-a-list, since most config settings like this are likely to be used in a single place, anyway.

Reply:
reworked as per previous comment.